<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-U7TEA4J" failures="1" tests="1" name="ThreeTest" time="312.71" errors="0" timestamp="2022-03-30T12:19:57 IST">
  <testcase classname="tests.ThreeTest" name="threeTest" time="300.022">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at tests.ThreeTest.threeTest(ThreeTest.java:23)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- threeTest -->
</testsuite> <!-- ThreeTest -->
